Formula/check.rb

```diff
@@ -3,7 +3,7 @@ class Check < Formula
   homepage "https://libcheck.github.io/check/"
   url "https://github.com/libcheck/check/releases/download/0.15.2/check-0.15.2.tar.gz"
   sha256 "a8de4e0bacfb4d76dd1c618ded263523b53b85d92a146d8835eb1a52932fa20a"
-  license "LGPL-2.1"
+  license "LGPL-2.1-or-later"
```

Comment:
A few files are licensed under GPL 3.0 or later, like https://github.com/libcheck/check/blob/master/m4/ax_create_stdint_h.m4

A few other older files are licensed under GPL 2.0 or later, like https://github.com/libcheck/check/blob/master/m4/ax_cflags_warn_all_ansi.m4

The project overall seems to be licensed under LGPL 2.1 or later.

Reply:
Is it appropriate to use all three license identifier (GPL 3.0 or later, GPL 2.0 or later and  LGPL-2.1-or-later) or just LGPL-2.1-or-later?

Reply:
It's probably safest to list both GPL 2.0 or later, as well as LGPL 2.1 or later (since GPL 3.0 falls under GPL 2.0 or later).